documentation/docs/libraries/nodejs/how_to/_01_get_node_info/_expected_output.md

```diff
@@ -0,0 +1,39 @@
+```json
+Node info:  {
+  nodeInfo: {
+    name: 'HORNET',
+    version: '2.0.0-beta.5',
+    status: {
+      isHealthy: true,
+      latestMilestone: [Object],
+      confirmedMilestone: [Object],
+      pruningIndex: 0
+    },
+    supportedProtocolVersions: [ 2 ],
+    protocol: {
+      version: 2,
+      networkName: 'testnet',
+      bech32Hrp: 'rms',
+      minPowScore: 1500,
+      rentStructure: [Object],
+      tokenSupply: '1450896407249092'
+    },
+    pendingProtocolParameters: [],
+    baseToken: {
+      name: 'Shimmer',
+      tickerSymbol: 'SMR',
+      unit: 'SMR',
+      subunit: 'glow',
+      decimals: 6,
+      useMetricPrefix: false
+    },
+    metrics: {
+      blocksPerSecond: 10.4,
+      referencedBlocksPerSecond: 10,
+      referencedRate: 96.15384615384616
+    },
+    features: []
+  },
+  url: 'https://api.testnet.shimmer.network'
+}
+```
```

Comment:
For this and most of other outputs, this is not JSON (hence why showing red on Github). Idk if docusaurus is going to render the colours well?

Reply:
yup, it's not JSON, but if I Docusuarus manages to colour in some variables this way.  

![image](https://user-images.githubusercontent.com/85233773/191506752-9ec1921d-6b48-41f2-83f3-059c0dbaf949.png)

I can switch them to plain text if you prefer. The result would be something like this: 

![image](https://user-images.githubusercontent.com/85233773/191507036-a6ba40d4-1083-4e63-a35d-21513d2f25be.png)